### DISCUSSION TITLE
Set language outside Epic launcher options?

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#66] [Q&A]
Is there any way to trigger that through the OpenCAGE settings or anything that could trigger a language change
outside of using Epic launcher options menus?

For speedrunning purposes people now use Italian because it's faster over all.
Alien Isolation has no internal "set language" options it is dependant on the Steam or Epic APIs.
In Steam this is easy.  You go to properties>set langage.  

In Epic setting a language is global to all games under that interface.   
This is fine if you're only using Epic for A:I but most have more games than that. 
And it's undesirable to change this globally back and forth too.

I'm guessing that Epic Store launcher sets up some register or flag and passes that to A:I.
Is there any way to read whatever is set and change it ? 

This issue has come up when people want to use Epic version to speedrun, but cannot use some of the tools that
work on Steam due to Epic's quirks such as this one.

**Comment by MattFiler** [comment]
There's actually an in-game way to set the audio language. I've tried enabling this on the Steam version and unfortunately although it remembers your preference, it doesn't appear to apply in-game. I'll test on the EGS version when I get chance and see if it functions there, it could be something from the Steam API overriding it.
